CONTRIBUTING.md

```diff
@@ -60,6 +60,10 @@ int main(int argc, char *argv[])
 
 Names should use underscores, not camel case, with class/struct names ending in `_t`.
 
+Headers should be included in the order `config.h`, C++ standard library headers,
+C library headers, Boost headers, and last osm2pgsql files. If the file is a .cpp
+file, include its .hpp file before all other headers.
```

Comment:
I don't think there is any reason to handle the own hpp file different. If at all, put it at the beginning of the osm2pgsql header block.